ledger/ledger-on-sql/BUILD.bazel

```diff
@@ -245,12 +245,15 @@ da_scala_test_suite(
 )
 
 [
+    # TODO https://github.com/digital-asset/daml/issues/8020
+    # Consider if we want to keep dev once exceptions are out of 1.dev
     conformance_test(
         name = "conformance-test-{}".format(db["name"]),
         lf_versions = [
             "stable",
             "latest",
             "preview",
+            "dev",
```

Comment:
I see you implemented already the fix for the TransactionCommitter, so this makes sense :+1: